lib/ansible/modules/source_control/gitlab_service.py

```diff
@@ -0,0 +1,400 @@
: {}, "username": {"required": 1}, "password": {"required": 1}, "jira_issue_transition_id": {}, "_events": ["commit", "merge_request"]}, "kubernetes": {"namespace": {"required": 1}, "api_url": {"required": 1}, "token": {"required": 1}, "ca_pem": {}}, "mattermost-slash-commands": {"token": {"required": 1}}, "slack-slash-commands": {"token": {"required": 1}}, "packagist": {"username": {"required": 1}, "token": {"required": 1}, "server": {}, "_events": ["push", "merge_request", "tag_push"]}, "pipelines-email": {"recipients": {"required": 1}, "notify_only_broken_pipelines": {"type": "bool"}, "_events": ["pipeline"]}, "pivotaltracker": {"token": {"required": 1}, "restrict_to_branch": {}, "_events": ["push"]}, "prometheus": {"api_url": {"required": 1}}, "pushover": {"api_key": {"required": 1}, "user_key": {"required": 1}, "priority": {"required": 1}, "device": {"required": 1}, "sound": {"required": 1}, "_events": ["push"]}, "redmine": {"new_issue_url": {"required": 1}, "project_url": {"required": 1}, "issues_url": {"required": 1}, "description": {}}, "youtrack": {"project_url": {"required": 1}, "issues_url": {"required": 1}, "description": {}}, "slack": {"webhook": {"required": 1}, "username": {}, "channel": {}, "notify_only_broken_pipelines": {"type": "bool"}, "notify_only_default_branch": {"type": "bool"}, "push_channel": {}, "issue_channel": {}, "confidential_issue_channel": {}, "merge_request_channel": {}, "note_channel": {}, "tag_push_channel": {}, "pipeline_channel": {}, "wiki_page_channel": {}, "push_events": {"type": "bool"}, "issues_events": {"type": "bool"}, "confidential_issues_events": {"type": "bool"}, "merge_requests_events": {"type": "bool"}, "note_events": {"type": "bool"}, "tag_push_events": {"type": "bool"}, "pipeline_events": {"type": "bool"}, "wiki_page_events": {"type": "bool"}}, "microsoft-teams": {"webhook": {"required": 1}}, "mattermost": {"webhook": {"required": 1}, "username": {}, "channel": {}, "notify_only_broken_pipelines": {"type": "bool"}, "notify_only_default_branch": {"type": "bool"}, "push_channel": {}, "issue_channel": {}, "confidential_issue_channel": {}, "merge_request_channel": {}, "note_channel": {}, "tag_push_channel": {}, "pipeline_channel": {}, "wiki_page_channel": {}, "push_events": {"type": "bool"}, "issues_events": {"type": "bool"}, "confidential_issues_events": {"type": "bool"}, "merge_requests_events": {"type": "bool"}, "note_events": {"type": "bool"}, "tag_push_events": {"type": "bool"}, "pipeline_events": {"type": "bool"}, "wiki_page_events": {"type": "bool"}}, "teamcity": {"teamcity_url": {"required": 1}, "build_type": {"required": 1}, "username": {"required": 1}, "password": {"required": 1}}, "github": {"token": {"required": 1}, "repository_url": {"required": 1}}, "jenkins": {"jenkins_url": {"required": 1}, "project_name": {"required": 1}, "username": {}, "password": {}}, "jenkins-deprecated": {"project_url": {"required": 1}, "pass_unstable": {"type": "bool"}, "multiproject_enabled": {"type": "bool"}}}
```

Comment:
This line should be "prettified" by adding indentation and line breaks, so that it properly fits within our line limits.